File: questao13/python/matematica.py
class Matematica:
    # Método estático para calcular o fatorial
    @staticmethod
    def fatorial(n):
        if n == 0 or n == 1:
            return 1
        else:
            return n * Matematica.fatorial(n - 1)


# Testando o método estático
print("Fatorial de 5:", Matematica.fatorial(5))
print("Fatorial de 7:", Matematica.fatorial(7))

File: questao13/java/Matematica.java
package questao13.java;



public class Matematica {

    // Método estático para calcular o fatorial
    public static int fatorial(int n) {
        if (n == 0 || n == 1) {
            return 1;
        } else {
            return n * fatorial(n - 1);
        }
    }

    public static void main(String[] args) {
        // Testando o método estático
        System.out.println("Fatorial de 5: " + Matematica.fatorial(5));
        System.out.println("Fatorial de 7: " + Matematica.fatorial(7));
    }
}
